index/templates/index/base.html

```diff
@@ -137,6 +137,11 @@
       </div>
       <div class="navbar-collapse collapse navbar-responsive-collapse">
         <ul class="nav navbar-nav">
+          <li><a href="{% url 'contest:archive' %}"><p class="p2">Contest</p></a></li>
+          <li><a href="{% url 'problem:volume' %}"><p class="p2">Problem</p></a></li>
+          <li><a href="{% url 'users:submit' %}"><p class="p2">Submit</p></a></li>
+          <li><a href="{% url 'status:status' %}"><p class="p2">Status</p></a></li>
+          <li><a href="{% url 'group:list' %}"><p class="p2">Group</p></a></li>
```

Comment:
You should modify origin code instead of add new code directly.
There are 2 contest, 2 problem, 2 submit, ... on the nav bar now, which is weird.